relayer/application_relayer.go

```diff
@@ -204,6 +204,7 @@ func (r *ApplicationRelayer) ProcessMessage(handler messages.MessageHandler) (co
 	if r.sourceWarpSignatureClient == nil {
 		signedMessage, err = r.signatureAggregator.CreateSignedMessage(
 			unsignedMessage,
+			make([]byte, 0),
```

Comment:
wasn't too clear to me what this `justification` field is meant for, and why it's always empty

Reply:
Justification field is part of [ACP-118](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/118-warp-signature-request/README.md) It's unused for now but is meant to be potentially extra data that would explain why  the data in the message field should be signed. It's unused for now though so that's why it's always empty

Reply:
Is there a reason we can't just pass in `nil` here? Allocating a slice that's never assigned to seems odd.